common-pg/src/main/scala/com/socrata/soql/analyzer/SoQLAnalyzerHelper.scala

```diff
@@ -56,10 +56,17 @@ object SoQLAnalyzerHelper {
     analyses match {
       case PipeQuery(l, r) =>
         val nl = remapAnalyses(columnIdMapping, l)
-        val prev = nl.outputSchemaLeaf
+        val prev = nl.outputSchema.leaf
```

Comment:
I'm guessing that this works the same as `nl.outputSchemaLeaf`?

Reply:
yes, i refactor soql-references and that is an ongoing process.